TheOracle2/UserContent/DiscordPlayerCharacterEntity.cs

```diff
@@ -24,12 +28,11 @@ public MessageComponent GetComponents() => new ComponentBuilder()
                 .WithButton("-Mo", $"lose-momentum-{Pc.Id}", row: 1, style: ButtonStyle.Secondary)
                 .WithButton("Burn", $"burn-momentum-{Pc.Id}", row: 0, style: ButtonStyle.Danger, emote: new Emoji("🔥"))
                 .WithButton("...", $"player-more-{Pc.Id}", row: 0, style: ButtonStyle.Primary).Build();
-
-        public string GetDiscordMessage()
+        public async Task<IMessage> GetDiscordMessage(IInteractionContext context)
         {
-            return null;
+            var channel = (Pc.ChannelId == context.Channel.Id) ? context.Channel : await (context.Client as DiscordSocketClient)?.Rest.GetChannelAsync(Pc.ChannelId) as IMessageChannel;
+            return await channel.GetMessageAsync(Pc.MessageId);
```

Comment:
You don't need to await the return here as you are already returning a Task<IMessage>

Reply:
hmm, here's what omnisharp complains about when i do just `return channel.GetMessageAsync(Pc.MessageId);` - is there something i'm missing here?
<img width="835" alt="Screen Shot 2022-01-25 at 5 12 40 PM" src="https://user-images.githubusercontent.com/5354757/151086397-bc25dab2-ddcd-440c-ab3a-07d0388f40a7.png">

Reply:
Huh, I might be a little confused on this one, and I did the review from GitHub web not an ide. This one was super minor anyway.